lms/djangoapps/teams/views.py

```diff
@@ -1087,3 +1116,38 @@ def delete(self, request, team_id, username):
             return Response(status=status.HTTP_204_NO_CONTENT)
         else:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class SearchResultPaginator(object):
```

Comment:
Actually, now that I have embraced that this paginator is completely independent of Teams, I think it belongs in /edx-platform/openedx/core/lib/api/paginators.py. That means you should try to write some unit tests for it (living over there also).